.github/workflows/inspektor-gadget.yml

```diff
@@ -587,6 +587,7 @@ jobs:
     strategy:
       matrix:
         type: [default, core]
+        driver: [none, docker]
```

Comment:
Does it mean that with this PR, we are now running the integration test in a multi-node minikube cluster (using docker driver) and also in a single-node cluster (using none driver)?